docs/Developers/Debug_Bears.rst

```diff
@@ -0,0 +1,77 @@
+Debug Bears
+===========
+
+This document provide a brief overview of coala's Debugging Interface.
+The Debugging Interface will help users to debug Bear code and step through
+it using pdb interface
+
+Usage of coala's Debugger
+=========================
+
+Command Line Interface
+----------------------
+
+Users can specify the bear they want to debug using ``--debug-bears``, i.e.
+
+.. code:: shell
+
+    $coala -b PEP8Bear,HelloWorldBear -f <filename> --debug-bears HelloWorldBear
+
+Below is the code given for a simple ``HelloWorldBear`` that prints a debug
+message for each file:
+
+.. code:: python
+
+    import logging
+
+    from coalib.bears.LocalBear import LocalBear
+
+
+    class HelloWorldBear(LocalBear):
+        def run(self,
+                filename,
+                file):
+            logging.info('Please follow newcomer guide.')
+            logging.debug('Hello World! Checking file {}.'.format(filename))
+
+After invoking the coala's debugger it will step into the ``run()`` method of a
+HelloWorldBear and help users to debug Bear code with passed settings.
```

Comment:
code term -> enclose in double backticks

Reply:
help**s**

Reply:
But actually, the "help users to debug Bear code" part. It's not only helping, it's debugging 100%ly :) I would omit the sentence. You can explain that it will step into your code regardless using `yield` or not.